packages/experimental/kai/src/containers/TaskList.tsx

```diff
@@ -156,14 +170,58 @@ export const TaskItem: FC<{
   onEnter?: (task: Task) => void;
   onDelete?: (task: Task) => void;
   onSave?: (task: Task) => void;
-}> = withReactor(({ task, readonly, showAssigned, onEnter, onDelete, onSave }) => {
+  isLast?: boolean;
+  orderIndex: number;
+}> = withReactor(({ task, readonly, showAssigned, onEnter, onDelete, onSave, orderIndex, isLast }) => {
   const { debug } = useAppState();
   useReactorContext({
     onChange: () => {
       onSave?.(task);
     }
   });
 
+  const onKeyDown = useCallback(
+    (e: KeyboardEvent<Element>) => {
+      switch (e.key) {
```

Comment:
Hey Will, thanks for fixing this.
Small note: we avoid single letter var names and try to consistently use `event` in these callback patterns, which should be consistent with our docs.

Reply:
Got it, will do!